spec/models/promulgated_rating_spec.rb

```diff
@@ -116,4 +109,25 @@
       )
     end
   end
+
+  context "#rating_profile" do
+    subject { rating.rating_profile }
+
+    context "when BGS throws a Share Error" do
+      let(:bgs) { Fakes::BGSService.new }
+
+      before do
+        allow(Fakes::BGSService).to receive(:new).and_return(bgs)
+        allow(bgs).to receive(:fetch_rating_profile)
+          .and_raise(BGS::ShareError, "Veteran does not meet the minimum disability requirements")
+        allow(bgs).to receive(:fetch_rating_profiles_in_range).and_call_original
+      end
+
+      it "Fetches the rating profile using RatingAtIssue" do
+        expect(subject.present?).to be true
```

Comment:
I recently learned this alternative:
```suggestion
        expect(subject.present?)
```

Reply:
Sweet, I am so excited to learn this!